subscriptions/logging.py

```diff
@@ -0,0 +1,9 @@
+import logging
+from functools import partial, partialmethod
+
+# add a high log level (higher than ERROR) which will store
+# information helpful for backtracing
+logging.ARCHIVE = logging.ERROR + 1
```

Comment:
Won't that put `sentry` on our heads?
Also, why not `logging.FATAL` if you want to go above everything.